Userland/Applications/Calculator/KeypadValue.cpp

```diff
@@ -169,3 +169,13 @@ void KeypadValue::set_to_0()
     m_value.set_to_0();
     m_decimal_places.set_to_0();
 }
+
+bool KeypadValue::operator<(i64 const rhs)
+{
+    return *this < BigInteger::SignedBigInteger::create_from(rhs);
+}
+
+bool KeypadValue::operator==(i64 const rhs)
+{
+    return *this == BigInteger::SignedBigInteger::create_from(rhs);
+}
```

Comment:
I think it would be far better to force the caller to make the conversion explicit, which was the case before.